data_generator/data_generator.go

```diff
@@ -397,17 +398,30 @@ func (state *GeneratorState) insertInitialIlkData(ilkId int64) error {
 		cat.InsertCatIlkLumpQuery,
 	}
 
+	contractAddressID, contractAddressErr := shared.GetOrCreateAddress(getRandomAddress(), state.db)
+	if contractAddressErr != nil {
+		return fmt.Errorf("error inserting random contract address: %w", contractAddressErr)
+	}
+
 	for _, intInsertSql := range intInsertions {
-		_, err := state.pgTx.Exec(intInsertSql, state.currentDiffID, state.currentHeader.Id, ilkId, rand.Int())
+		// HACK because the cat queries now take an address_id. If the other storage queries start taking an
+		// address_id, remove this hack
+		var err error
+		if intInsertSql == cat.InsertCatIlkChopQuery || intInsertSql == cat.InsertCatIlkLumpQuery {
```

Comment:
I guess we never deal with `cat_ilk_flip` in here?

Reply:
Oddly no, but I guess since it's test data.